docs/advanced/api/authentication.rst

```diff
@@ -0,0 +1,63 @@
+==============
+Authentication
+==============
+
+The Flocker REST API uses TLS certification to secure and authenticate requests.
```

Comment:
Just "TLS" rather than "TLS certification". The latter sounds like a required qualification!